libs/scir/src/lib.rs

```diff
@@ -131,6 +131,13 @@ impl From<Slice> for SignalId {
     }
 }
 
+impl From<SliceOne> for SignalId {
```

Comment:
Why is this implemented? Shouldn't if you're losing information...

Reply:
So you can call expose port on a Slice or SliceOne.